src/core/reactor.cc

```diff
@@ -4288,6 +4288,20 @@ unsigned smp::adjust_max_networking_aio_io_control_blocks(unsigned network_iocbs
     return network_iocbs;
 }
 
+future<> join_memory_prefault() {
+    auto& r = engine();
+    if (!r._smp->memory_prefault_initialized()) {
+        seastar_logger.warn("Memory prefaulter is not initialized but joined");
```

Comment:
This warning isn't helpful to users; what can they do?

Reply:
The application doesn't know if the user wants to prefault memory or not. It's common in local testing not to prefault, and in production to prefault.